biome.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
I already used biome once, I used the following config: https://github.com/kibertoad/unzipomatic/blob/main/biome.json

Reply:
I'm using the same configuration we are using on `amaro` and `nodejs-loaders`.